core/iwasm/libraries/debug-engine/gdbserver.c

```diff
@@ -98,6 +98,14 @@ wasm_launch_gdbserver(char *host, int port)
         goto fail;
     }
 
+    socklen = sizeof(addr);
+    if (getsockname(listen_fd, (void *)&addr, &socklen) == -1) {
+        LOG_ERROR("%s", strerror(errno));
+        goto fail;
+    }
+
+    LOG_VERBOSE("Listening on %s:%d\n", host, ntohs(addr.sin_port));
```

Comment:
Suggest to use LOG_WARNING() instead so user can always see the port number without adding "-v=5" to iwasm, or he won't be able to know the port number without "-v=5".

Reply:
Done